assignments/function-conversion.js

```diff
@@ -5,26 +5,56 @@
 // };
 // myFunction();
 
+let myFunction = () => {
+    console.log("Function was invoked!");
+}
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {
+    return param;
+}
+
+console.log(anotherFunction("example"));
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1, param2) => {
+    return param1 + param2;
+}
+
+console.log(add(1,2));
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+let subtract = (param1, param2) => {
+    return param1 - param2;
+}
+
+console.log(subtract(1,2));
+
 
 // Stretch
 
-// exampleArray = [1,2,3,4];
-// const triple = exampleArray.map(function (num) {
+exampleArray = [1,2,3,4];
+// const triple = exampleArray.map(function (num) { num * 3 } )
+
 //   return num * 3;
+
 // });
-// console.log(triple);
\ No newline at end of file
+
+// console.log(triple);
+
+let triple = exampleArray.map((num) => {return num * 3;});
+
```

Comment:
Great job solving this one with `.map`. since you are using fat arrow syntax you actually don't need the brackets or the `return`. Fat arrow functions have what is called an implicit return.  You could write the `map` as such 

`.map((num) => num * 3);`